force-app/main/default/lwc/participantAdder/participantAdder.html

```diff
@@ -0,0 +1,42 @@
+<template>
+    <header class="slds-modal__header">
+        <h2 id="modal-heading-01" class="slds-modal__title slds-hyphenate">
+            {labels.addServiceParticipants}
+        </h2>
+    </header>
+
+    <template if:true={isLoaded}>
+        <div class="slds-modal__content slds-p-around_medium" id="modal-content-id-1">
+            <c-participant-selector
+                service-id={serviceId}
+                schedule-id={recordId}
+                previously-selected-participant-contact-ids={selectedParticipantContactIds}
```

Comment:
This is a mouthful lol, what do you think of something like existing participants or existing contact ids?